Data/Ascii/AsciiReader.cpp

```diff
@@ -6,14 +6,16 @@
 #include <sstream>
 #include <utility>
 
-#include "AsciiReader.hpp"
+#include "Data/Ascii/AsciiReader.hpp"
```

Comment:
You could rename the files to AsciiIO, because you also wrote a write function now.

Reply:
Would agree better with `RootDataIO`. But the three I's are also ugly.

Reply:
Ok, up to you

Reply:
Renamed to `AsciiDataIO` in accordance with `RootDataIO`